### DISCUSSION TITLE
FLSun V400 Mainboard change...

### DISCUSSION THREAD

**Discussion by Megamanwwh** [Guilouz/Klipper-Flsun-Speeder-Pad#93] [Q&A]
Hello, I have a FLSun V400 here and was wondering if I could exchange the existing Board Nano Ver 2.1 for the 3.1 and use it with this customized software. The actual board is currently not available and the 3.1 looks like it would fit without any problems.

[](https://drive.google.com/file/d/1sKB5YFyslCanC_KfMRq6I4a0pCcIQ8WC/view?usp=sharing)

**Comment by kaktuss77** [comment]
hi, 

I have use Nano 2.0 config files for my 2.1 and it's work. (maybe luck?) 

Regards
